buildkite/scripts/release/manager.sh

```diff
@@ -406,19 +431,21 @@ function publish_debian() {
                 --new-version ${__target_version} \
                 --suite "unstable" \
                 --new-suite ${__channel} \
-                --new-name ${__new_artifact_name}
+                --new-name ${__new_artifact_name} \
+                --arch ${__arch}
     fi
 
     echo " 🍥  Publishing $__artifact debian to $__channel channel with $__target_version version"
-    echo "     📦  Target debian version: $(calculate_debian_version $__artifact $__target_version $__codename "$__network" )"
+    echo "     📦  Target debian version: $(calculate_debian_version $__artifact $__target_version $__codename "$__network" "$__arch")"
```

Comment:
Here as well. I think the reason for quoting is they might be empty string? One more reason to ditch bash

Reply:
bash linter suggest this change, so i followed. Yes it might be an empty string or has space or glob, That will prevent it. In this case it won't happen but this is common approach